### PR TITLE
#92 [phpcs] Apply coding standard changes

### PR DESCRIPTION
**Pull request by pritalpatel**
Thanks @yvesh for suggestions.
Send Coding standard changes for https://github.com/joomla-projects/joomla-browser/issues/92

### CODE REVIEW

File: src/JoomlaBrowser.php [hunk 1]
```diff
@@ -110,191 +128,212 @@ public function doFrontendLogout()
...
+
+		// @todo: installation of sample data needs to be created
+
+		/*if ($this->config['install sample data']) :
+		    $this->debug('I install Sample Data:' . $this->config['sample data']);
+		    $I->selectOption('#jform_sample_file', $this->config['sample data']);
+		else :
+		    $this->debug('I install Joomla without Sample Data');
+		    $I->selectOption('#jform_sample_file', '#jform_sample_file0'); // No sample data
+		endif;*/
```
Comment (yvesh) [inline src/JoomlaBrowser.php]: Don't leave any commented code.

https://medium.com/@kentcdodds/please-don-t-commit-commented-out-code-53d0b5b26d5f#.n7t3shymb

File: src/JoomlaBrowser.php [hunk 2]
```diff
@@ -886,15 +970,17 @@ public function verifyAvailableTabs($expectedTabs, $tabsLocator = ['xpath' => "/
...
     {
         $I = $this;
-        $this->debug('I click on never');
-        $I->waitForElement(['link' => 'Never'], 60);
-        $I->click(['link' => 'Never']);
+		$this->debug('I click on never');
+		$I->waitForElement(['link' => 'Never'], 60);
+		$I->click(['link' => 'Never']);
     }
 }
```
Comment (yvesh) [inline src/JoomlaBrowser.php]: Add a newline after the closing }

**Comment by yvesh** [comment]
Hi @pritalpatel, your PR has one big problem. Your tab is 8 spaces long and not 4. Can you adjust that please? Thank you :)

File: src/JoomlaBrowser.php (hunk 1 above)
Comment (pritalpatel) [inline src/JoomlaBrowser.php]: I leave it because it says `@todo` there. I will remove it.

**Comment by pritalpatel** [comment]
Thanks @yvesh Actually that problem with 8 char was because of I have fixed conflict with meld tool. Now, I have corrected it from Sublime Text. So, should be fine now.

**Comment by javigomez** [comment]
closing for #116

**Pull request closed without merging**